src/test/util.rs

```diff
@@ -494,7 +484,16 @@ pub(crate) async fn drop_collection<T>(coll: &Collection<T>)
 where
     T: Serialize + DeserializeOwned + Unpin + Debug + Send + Sync,
 {
-    match coll.drop().await.map_err(|e| *e.kind) {
+    drop_collection_opt(coll, |a| a).await
+}
+
+pub(crate) async fn drop_collection_opt<'a, T>(
```

Comment:
Just happened to notice that we already suppress namespace not found errors (i.e. code 26, RUST-392) in our drop collection implementation, so this wrapper shouldn't be necessary at all anymore

Reply:
Oh, nice.  Removed!